### DISCUSSION TITLE
IMSAI8080 talking with another computers via acoustic coupler

### DISCUSSION THREAD

**Discussion by elfuska** [thehighnibble/thehighnibble.github.io#95] [Show and tell]
Hi all. 

  Just wanted to share with you some videos involving my IMSAI8080 replica communicating with another systems via old acoustic couplers:

  In [the first one](https://fediverse.tv/w/5VbJh7kyWUfX8Cm3uV44aM) you can see the IMSAI connecting to a Linux computer via an old acoustic coupler attached to an analog phone. In the video description you can check the setup.

  The [second video](https://fediverse.tv/w/k8EKpPvWZ47qTBDddVKrb1) shows the IMSAI receiving calls using the acoustic coupler. In this case the caller party is the Linux machine attached to an old modem. I dial the IMSAI extension number, the phone rings, and I attach the phone to the acoustic coupler connected to the IMSAI.

  The [last one](https://fediverse.tv/w/oF2w5UJH8EYX1AD86HWhaj) will bring back memories for any of you who know about blue boxing. With [blue boxing](https://en.wikipedia.org/wiki/Blue_box) you could send tones to the phone line making the trunk think you were another trunk, and that basically meant getting free calls. Back in the time of the BBSs, when you had to pay for every single call, this was relatively common. Now blue boxing no longer works, but sites like [ProjectMF](https://projectmf.org/) allows you to setup a local environment where you can legally and locally have fun using your blue box. In the video I set up a connection between a Linux machine and the IMSAI using two acoustic couplers and a blue box.

  Hope you enjoy the videos. Thank you for all the information you post here and of course thanks to @dmcnaugh for both the IMSAI and the S-132 kits, I've been having fun with them since the very first day I received them.

  Regards.

**Comment by donfroula** [comment]
Well that is cool!

I'm the guy who built and operates ProjectMF. Coincidentally, my IMSAI and S132 just arrive from Australia. Very cool to see the IMSAI routed through my switch! I'll have to try this!

Oh, just realized this is your local machine. How did you get ProjectMF to build?! You must be incredibly patient, installing the ancient Debian distro and getting it to compile. Not to mention getting the Ethernet hardware running.

Did you duplicate my dial plan? Neat.

Best,

Don

**Comment by elfuska** [comment]
Wow, hello Don! So happy to hear you liked it!

Before trying to build my own ProjectMF server, I tested my blue box (well, in fact is your blue box) with your server. I hope you won't try to kill me but... do you remember a call from some assh*** with a bad english accent asking something about Meryland? Well... I was dialing some random extensions and one of them was the inward operator extension. I was so shocked to hear someone (I guess it was you) on the other side that, instead of instantly hanging up the phone, my mind was like "try not to look too stupid". And, I don't know why, I asked "Am I calling to Meryland?". Sorry Don, I just hope I didn't wake you up :)

Building projectMF was hard. First finding the right workstation. Then finding the old debian installation media. When the OS was installed some hardware was too modern to be supported by such an old kernel (specially the network cards), so I had to modify and recompile the drivers. Compiling asterisk and zaptel was more or less painless thanks to the packages provided. And thanks to the examples and the documentation I was able to set up two extensions for the two analog phones connected to the ATA, and a couple of extensions pointing to some old spanish phreaking scene recordings (you can hear one of them in [this video](https://fediverse.tv/w/phxWeYsVbBx8yjoBpJpbTm), although I made a mistake dialing). But overall I had a lot of fun and I learnt a lot about different topics while setting everything up. So a big thank you Don for your projects. People like you, like David (the creator of this kit), like Udo (the creator of z80pack) are bringing us so much joy.

I'm sure you will love this little computer. Eager to see your next projects with it.

Kind regards.

**Reply by donfroula** [reply]
I know of only one other who was able to follow my convoluted Asterisk patching instructions. Nice job! 

I love your build of the Arduino Blue Box. Very sleek. I just finished a working breadboard of Steve Wozniak's blue box based upon his old hand-drawn schematics. It works very well.

https://youtu.be/Ow_82ffnkhk?

I don't recall your call exactly, but I always enjoy calls from users of the system.

Can't wait to get started on the IMSAI kit this weekend. My fresh flux pen just arrived.
